genesyscloud/resource_genesyscloud_auth_division_test.go

```diff
@@ -165,3 +193,30 @@ func validateHomeDivisionID(divResourceName string) resource.TestCheckFunc {
 		return nil
 	}
 }
+
+func cleanupAuthDivision(idPrefix string) {
+	authAPI := platformclientv2.NewAuthorizationApi()
+
```

Comment:
I noticed here - [GET /api/v2/authorization/divisions](https://developer.genesys.cloud/devapps/api-explorer#get-api-v2-authorization-divisions) - that the objects api is used to get all divisions. 

Also, you can pass `sdkConfig` to `NewObjectsApiWithConfig` without calling AuthorizeSdk because the sdkConfig variable should already exist in the genesyscloud package and be authorized

Reply:
Added sdkConfig